inc/storefront-template-functions.php

```diff
@@ -159,14 +159,20 @@ function storefront_header_widget_region() {
 	 * @return void
 	 */
 	function storefront_site_branding() {
-		if ( function_exists( 'the_custom_logo' ) && has_custom_logo() ) {
-			the_custom_logo();
+    if ( function_exists( 'the_custom_logo' ) && has_custom_logo() ) {
+      $logo = get_custom_logo();
+
+      echo $logo = is_home() ? '<h1>' . $logo . '</h1>' : $logo;
```

Comment:
@jameskoster 

> If the h1 wrapper is preferred, let's just tweak storefront_site_branding() so that that site-branding wrapping div also wraps the logo. We can then use that class for all layout purposes. Then it should just be a case of removing the logo specific styles which is easy enough.

Yes, the `<h1>` is great on home page.
When it wraps the image, its content will be the `alt` attribute's value.

Seems a better idea to me also.
Since it's kinda CSS related, I let you deal with that one ?